benchmarks/dynamo/ci_expected_accuracy/inductor_torchbench_training.csv

```diff
@@ -114,7 +114,7 @@ hf_Reformer,pass,23
 
 
 
-hf_Roberta_base,fail_accuracy,6
```

Comment:
I can not repro the failure locally. Also from the CI logs, it does not show which tensor has violates the tolerance. It may not be real accuracy problem